packages/vendure-plugin-popularity-scores/src/popularity-scores.service.ts

```diff
@@ -141,12 +143,31 @@ export class SortService implements OnModuleInit {
 
       const uniqueProductIds = [...new Set(productIds)];
       if (uniqueProductIds.length) {
-        const summedProductsValue = await productSummingQuery
-          .andWhere('product.id IN (:...ids)', { ids: uniqueProductIds })
-          .getRawOne();
+        var sliceArray = (arr: number[], chunkSize: number): number[][] => {
```

Comment:
Please use let or const instead of var. Also, can we move the sliceArray function into a util instead of having it inline in the function?